frontend/src/pages/admin/project-list.js

```diff
@@ -44,216 +60,177 @@ function projectToRow (project, onDeleteClick) {
+  document.title = 'Projects - Administration | Ibutsu';
+  return (
+    <React.Fragment>
+      <PageSection id="page" variant={PageSectionVariants.light}>
+        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+          <Flex>
+            <FlexItem spacer={{ default: 'spacerLg' }}>
+              <TextContent>
+                <Text className="title" component="h1" ouiaId="admin-projects">Projects</Text>
+              </TextContent>
+            </FlexItem>
           </Flex>
-        </PageSection>
-        <PageSection className="pf-u-pb-0">
-          <Card>
-            <CardBody className="pf-u-p-0">
-              <FilterTable
-                columns={columns}
-                rows={rows}
-                filters={filters}
-                pagination={pagination}
-                isEmpty={this.state.isEmpty}
-                isError={this.state.isError}
-                onSetPage={this.setPage}
-                onSetPageSize={this.setPageSize}
-              />
-            </CardBody>
-          </Card>
-        </PageSection>
-        <Modal
-          title="Confirm Delete"
-          variant="small"
-          isOpen={this.state.isDeleteModalOpen}
-          onClose={this.onDeleteModalClose}
-          actions={[
-            <Button
-              key="delete"
-              variant="danger"
-              ouiaId="admin-projects-modal-delete"
-              isLoading={this.state.isDeleting}
-              isDisabled={this.state.isDeleting}
-              onClick={this.onModalDeleteClick}
-            >
-              {this.state.isDeleting ? 'Deleting...' : 'Delete'}
-            </Button>,
-            <Button
-              key="cancel"
-              variant="secondary"
-              ouiaId="admin-projects-modal-cancel"
-              isDisabled={this.state.isDeleting}
-              onClick={this.onDeleteModalClose}
-            >
-              Cancel
-            </Button>
-          ]}
-        >
-          Are you sure you want to delete &ldquo;{this.state.selectedProject && this.state.selectedProject.title}&rdquo;? This cannot be undone!
-        </Modal>
-      </React.Fragment>
-    );
-  }
-}
+          <Flex>
+            <FlexItem>
+              <Button
+                aria-label="Add project"
+                variant="secondary"
+                title="Add project"
+                ouiaId="admin-projects-add"
+                component={(props) => <Link {...props} to="/admin/projects/new" />}
+              >
+                <PlusCircleIcon /> Add Project
+              </Button>
+            </FlexItem>
+          </Flex>
+        </Flex>
+      </PageSection>
+      <PageSection className="pf-u-pb-0">
+        {!isEmpty &&
+        <Card>
+          <CardBody className="pf-u-p-0">
+            <FilterTable
+              columns={COLUMNS}
+              rows={filteredProjects ? filteredProjects.map((p) => projectToRow(p)) : [getSpinnerRow(4)]}
+              activeFilters={activeFilters}
```

Comment:
To make it work correctly we also need to provide `onRemoveFilter={onRemoveFilter}` that will handle chip removing logic. Otherwise when you click on delete chip - page crashes.

onRemoveFilter will need to look like

```
  const onRemoveFilter = () => {
    setFilterText('');
    setActiveFilters({});
    setIsEmpty(false);
    setFilteredProjects(projects);
    setTotalItems(projects.length);
  };
  ```

Reply:
AH! didn't remove filter in my testing - good catch, adding.